sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/polling/SyncOperationLocationPollingStrategy.java

```diff
@@ -0,0 +1,134 @@
+import com.azure.core.http.HttpMethod;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.polling.LongRunningOperationStatus;
+import com.azure.core.util.polling.PollResponse;
+import com.azure.core.util.polling.PollingContext;
+import com.azure.core.util.polling.PollingStrategyOptions;
+import com.azure.core.util.polling.SyncOperationResourcePollingStrategy;
+import com.azure.core.util.serializer.ObjectSerializer;
+import com.azure.core.util.serializer.TypeReference;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+
+/**
+ * Implements a synchronous operation location polling strategy, from Operation-Location.
+ *
+ * @param <T> the type of the response type from a polling call, or BinaryData if raw response body should be kept
+ * @param <U> the type of the final result object to deserialize into, or BinaryData if raw response body should be
+ *           kept
+ */
+public class SyncOperationLocationPollingStrategy<T, U> extends SyncOperationResourcePollingStrategy<T, U> {
+
+    private static final ClientLogger LOGGER = new ClientLogger(SyncOperationLocationPollingStrategy.class);
+
+    private static final HttpHeaderName OPERATION_LOCATION_HEADER
+        = HttpHeaderName.fromString("Operation-Location");
+
+    private static final TypeReference<PostPollResult> POST_POLL_RESULT_TYPE_REFERENCE
+        = TypeReference.createInstance(PostPollResult.class);
+
+    private final ObjectSerializer serializer;
+    private final String endpoint;
+
+    /**
+     * Creates an instance of the operation resource polling strategy.
+     *
+     * @param pollingStrategyOptions options to configure this polling strategy.
+     * @throws NullPointerException if {@code pollingStrategyOptions} is null.
+     */
+    public SyncOperationLocationPollingStrategy(PollingStrategyOptions pollingStrategyOptions) {
+        super(OPERATION_LOCATION_HEADER, pollingStrategyOptions);
+        this.endpoint = pollingStrategyOptions.getEndpoint();
+        this.serializer = pollingStrategyOptions.getSerializer() != null
+            ? pollingStrategyOptions.getSerializer() : new DefaultJsonSerializer();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PollResponse<T> onInitialResponse(Response<?> response, PollingContext<T> pollingContext,
+                                             TypeReference<T> pollResponseType) {
+        HttpHeader operationLocationHeader = response.getHeaders().get(OPERATION_LOCATION_HEADER);
+        if (operationLocationHeader != null) {
+            pollingContext.setData(OPERATION_LOCATION_HEADER.getCaseSensitiveName(),
+                PollingUtils.getAbsolutePath(operationLocationHeader.getValue(), endpoint, LOGGER));
+        }
+        final String httpMethod = response.getRequest().getHttpMethod().name();
+        pollingContext.setData(PollingConstants.HTTP_METHOD, httpMethod);
```

Comment:
Yes, in long term, we may.

`operation-id` would also need that we provide the polling URL (or the actual polling method) to the strategy, which is more complex in core.

Hence since it is equivalent, we still use the `operation-location`.